demo/blocks_interpretation/run.py

```diff
@@ -0,0 +1,57 @@
+import gradio as gr
```

Comment:
Adding this demo even though it's not used in the guide because I think it will be a cool thing to show once we have the "all demos in one place" page (#1638) and could also be an interesting test in the meantime. 

The reason it's not used in the guide is because it's a tad long to include all in one block. I opted to break it up into chunks and explain each chunk separately.

Reply:
Agree with testing. Interpretation is also a good thing to include in integration testing @dawoodkhan82